src/bootstrap.js

```diff
@@ -14,10 +18,25 @@ function install() {} // eslint-disable-line no-unused-vars
 function uninstall() {} // eslint-disable-line no-unused-vars
 
 function startup() { // eslint-disable-line no-unused-vars
+  Services.prefs.setBoolPref(UA_ENABLE_PREF_NAME, true);
```

Comment:
We've discussed this in Taipei. :) The preference should reset on every browser reboot. If someone complains, we can change this in the future, but for now, we want to have the pref forced on on a browser startup.